src/pratt.rs

```diff
@@ -252,11 +273,11 @@ impl<'a, I, O, E, Atom, InfixOps, InfixOpsOut, PostfixOps, PostfixOpsOut>
     }
 }
 
-type InfixBuilder<E> = fn(lhs: E, rhs: E) -> E;
+type InfixBuilder<Op, O> = fn(op: Op, children: [O; 2]) -> O;
 
-type PrefixBuilder<E> = fn(rhs: E) -> E;
+type PrefixBuilder<Op, O> = fn(op: Op, child: O) -> O;
 
-type PostfixBuilder<E> = fn(rhs: E) -> E;
+type PostfixBuilder<Op, O> = fn(op: Op, child: O) -> O;
```

Comment:
This is one of the points that I was unsure about. I think I would prefer and find it more natural for `PostfixBuilder` to be `fn(O, Op) -> O` because the op follows the expression when parsing.